### DISCUSSION TITLE
7 segment digital clock code problem

### DISCUSSION THREAD

**Discussion by sulman3329** [community/community#144119] [New to GitHub]
Facing problem coding Arduino segment digital clock 
Unable to install clock master library 
And code

**Comment by sulman3329** [comment]
![Screenshot_20241110-191919](https://github.com/user-attachments/assets/959313db-1d87-4c21-a081-cd795d49cc11)
![Screenshot_20241110-191835](https://github.com/user-attachments/assets/e7e82d1f-201b-495b-9297-94b711bd225b)

**Comment by ghostinhershell** [comment]
Hi @sulman3329,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
